_data/navbars/about.yml

```diff
@@ -7,6 +7,8 @@ section:
   path: /about/about-calico
 - title: Networking
   path: /about/about-networking
+- title: eBPF
+  path: /about/about-ebpf
```

Comment:
Pondering whether to suggest moving this a little later in the nav order. e.g. If you don't already know about k8s networking, then some of the things discussed in this guide won't mean much.